### DISCUSSION TITLE
Border not visible in Custom nodes which are source and target Nodes using <Handle>

### DISCUSSION THREAD

**Discussion by dheerajmishradv** [xyflow/xyflow#4017] [Q&A]
I have requirement where all nodes are Custom nodes
I created the custom nodes and defined <Handle> in them
For starting node and ending node I am not able to see borders whereas borders are visible for in between nodes

**Comment by bcakmakoglu** [comment]
Well... if you're using custom nodes did you actually define any styles for them?
The library isn't magically going to add styles for nodes it doesn't know anything about so you have to style your own nodes yourself.

For example:
```css
.react-flow__node-custom {
	padding: 1rem;
	border: 1px solid hotpink;
	background: #fff;
	color: #000;
}
